sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/DoNotCatchNullReferenceException.cs

```diff
@@ -0,0 +1,20 @@
+using System;
```

Comment:
Checking for null is not always possible - for example a 3rd party library may be throwing.
Do we want to allow cases when exception is being caught, logged and rethrown?

Regardless of decision, need to add a test for that.